docs/integrations/pytorch_lightning.rst

```diff
@@ -3,719 +3,182 @@
-* ``pytorch-lightning==1.0.7`` or newer. See |lightning-install|.
-
-You also need minimal familiarity with the PyTorch Lightning. Have a look at the "|lightning-guide|" guide to get started.
-
-Step 1: Import Libraries
-^^^^^^^^^^^^^^^^^^^^^^^^
-Import necessary libraries.
-
-.. code-block:: python3
-
-    import os
-
-    import torch
-    from torch.nn import functional as F
-    from torch.utils.data import DataLoader
-    from torchvision.datasets import MNIST
-    from torchvision import transforms
-
-    import pytorch_lightning as pl
-
-Notice ``pytorch_lightning`` at the bottom.
-
-Step 2: Define Hyper-Parameters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Define Python dictionary with hyper-parameters for model training.
-
-.. code-block:: python3
-
-    PARAMS = {'max_epochs': 3,
-              'learning_rate': 0.005,
-              'batch_size': 32}
-
-This dictionary will later be passed to the Neptune logger (you will see how to do it in :ref:`step 4 <create-neptune-logger>`), so that you will see hyper-parameters in experiment `Parameters` tab.
-
-Step 3: Define LightningModule and DataLoader
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Implement minimal example of the ``pl.LightningModule`` and simple ``DataLoader``.
-
-.. code-block:: python3
-
-    # pl.LightningModule
-    class LitModel(pl.LightningModule):
-        def __init__(self):
-            super().__init__()
-            self.l1 = torch.nn.Linear(28 * 28, 10)
-
-        def forward(self, x):
-            return torch.relu(self.l1(x.view(x.size(0), -1)))
 
-        def training_step(self, batch, batch_idx):
-            x, y = batch
-            y_hat = self(x)
-            loss = F.cross_entropy(y_hat, y)
-            self.log('train_loss', loss)
-            return loss
-
-        def configure_optimizers(self):
-            return torch.optim.Adam(self.parameters(), lr=PARAMS['learning_rate'])
-
-    # DataLoader
-    train_loader = DataLoader(MNIST(os.getcwd(), download=True, transform=transforms.ToTensor()),
-                              batch_size=PARAMS['batch_size'])
-
-Few explanations here:
-
-* Cross entropy logging is defined in the ``training_step`` method in this way:
-
-.. code-block:: python3
-
-    self.log('train_loss', loss)
-
-This loss will be logged to Neptune during training as a ``train_loss``. You will see it in the Experiment's `Charts` tab (as "train_loss" chart) and `Logs` tab (as raw numeric values).
-
-* ``DataLoader`` is a pure PyTorch object.
-* Notice, that you pass ``learning_rate`` and ``batch_size`` from the ``PARAMS`` dictionary - all params will be logged as experiment parameters.
-
-.. _create-neptune-logger:
-
-Step 4: Create NeptuneLogger
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Instantiate ``NeptuneLogger`` with necessary parameters.
+Create NeptuneLogger
+^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: python3
 
-    from pytorch_lightning.loggers.neptune import NeptuneLogger
+    from neptune.new.integrations.pytorch_lightning import NeptuneLogger
```

Comment:
This should be the neptune-contrib logger.

Reply:
fixed